skin/src/main/res/values/strings.xml

```diff
@@ -139,7 +139,10 @@
     <string name="rss_local_error_load_task">Unable to load the selected task</string>
     <string name="rss_error_select_answer">Please select an answer</string>
     <string name="rss_error_invalid_password">Invalid Password</string>
+    <string name="rss_error_small_password">%1$d characters minimum</string>
     <string name="rss_error_invalid_email">Invalid Email</string>
+    <string name="rss_repeat_signup">Your account has been signed out, you need to repeat the sign up.</string>
```

Comment:
in the case of automatically-generated passwords, as users don't hold the password, sign-out actually means that they will need to re-register.